first-party/jib-layer-filter-extension-maven/src/test/java/com/google/cloud/tools/jib/maven/extension/layerfilter/JibLayerFilterExtensionTest.java

```diff
@@ -47,6 +60,23 @@
 
   @Mock private Configuration config;
   @Mock private ExtensionLogger logger;
+  @Mock private MavenData mavenData;
+  @Mock private MavenProject mavenProject;
+  @Mock private MavenProject mavenParentProject;
+  @Mock private MavenSession mavenSession;
+  @Mock private ProjectDependenciesResolver projectDependenciesResolver;
+  @Mock private DependencyResolutionResult dependencyResolutionResult;
+
+  @Before
+  public void setUp() throws DependencyResolutionException {
+    when(config.getFilters()).thenReturn(Collections.emptyList());
+    when(config.isCreateParentDependencyLayers()).thenReturn(false);
```

Comment:
nit: Mockito by default returns false, so this is unnecessary.

Reply:
ok